File: app/src/main/java/com/legue/axel/lolsummonertool/ProfilSuggestionProvider.java
package com.legue.axel.lolsummonertool;

import android.content.SearchRecentSuggestionsProvider;

public class ProfilSuggestionProvider extends SearchRecentSuggestionsProvider {
    public final static String AUTHORITY = "com.legue.axel.lolsummonertool.ProfilSuggestionProvider";
    public final static int MODE = DATABASE_MODE_QUERIES | DATABASE_MODE_2LINES;

    public ProfilSuggestionProvider() {
        setupSuggestions(AUTHORITY, MODE);
    }
}

File: app/src/main/java/com/legue/axel/lolsummonertool/ProfilSuggestionProvider.kt
package com.legue.axel.lolsummonertool

import android.content.SearchRecentSuggestionsProvider

class ProfilSuggestionProvider : SearchRecentSuggestionsProvider() {

    companion object {
        const val AUTHORITY = "com.legue.axel.lolsummonertool.ProfilSuggestionProvider"
        const val MODE = DATABASE_MODE_QUERIES or DATABASE_MODE_2LINES
    }

    init {
        setupSuggestions(AUTHORITY, MODE)
    }


}
